common/changes/focuszone-donotfocus_2017-01-06-02-46.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Added an attribute to prevent focuszone focus",
```

Comment:
Please format as "FocusZone: Added data-not-focusable attribute to prevent focus"